src/conf.lua

```diff
@@ -35,7 +35,7 @@ function love.conf(t)
 	t.modules.thread = true              -- Enable the thread module (boolean)
 
 	for _, flag in ipairs(arg) do
-		if flag == "--unit-tests" or flag == "--headless" then
+		if flag == "--unit-tests" or flag == "--headless" or flag == "--help" then
 			local stub = require("tests/stub")
 			t.window, t.modules.window, t.modules.graphics, t.modules.audio = false, false, false, false
```

Comment:
I snuck in a bonus fix: `--help` will no longer open the main window briefly; it counts as a headless mode like `--unit-tests` and `--headless`.